client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -150,18 +166,21 @@ export default React.createClass( {
 	},
 
 	getUpdateWarning() {
-		const newVersions = this.getAvailableNewVersions();
+		const newVersions = getAvailableNewVersions( this.props );
 		if ( newVersions.length > 0 ) {
 			if ( this.props.selectedSite ) {
+				const textVersion = this.translate( 'Version' );
+				const textIsAvailable = this.translate( 'is available' );
+
 				return (
 					<Notice
 						status="is-warning"
 						className="plugin-meta__version-notice"
 						showDismiss={ false }
 						icon="sync"
-						text={ i18n.translate( 'A new version is available.' ) }>
+						text={ `${ textVersion } ${ newVersions[ 0 ].newVersion } ${ textIsAvailable }` }>
```

Comment:
Is there a reason why this can't be translated like 
```
i18n.translate( 'Version %(newPluginVersion)s is available', { args: { newPluginVersion: newVersions[ 0 ].newVersion } } )
```